orgm/orgm.model.lkml

```diff
@@ -5,76 +5,100 @@ include: "/orgm/orgm_views/staging/*.view.lkml"
 fiscal_month_offset: -11
 
 
-explore: delete_history {
+explore: account {
   group_label: "zHeroku Postgres OrgM"
-}
 
-explore: hc_object_status {
-  group_label: "zHeroku Postgres OrgM"
-}
-explore: opportunity {
-  group_label: "zHeroku Postgres OrgM"
+  join: opportunity {
+    sql_on: ${account.sfid} = ${opportunity.accountid} ;;
+    relationship: many_to_one
```

Comment:
`relationship: one_to_many`
This is from account to opportunity. Relationship is always base explore to join.